packages/tools/src/account-cli.ts

```diff
@@ -54,6 +54,9 @@ program
     "Eth address (layer2 lock args, using --private-key value to calculate if not provided)"
   )
   .option("-c, --capacity <capacity>", "capacity in shannons", "40000000000")
+  .option("-b, --bip44", 'whether to use BIP44 wallet address', false)
```

Comment:
Any problem with fill child private key in `--private-key` ?

Reply:
Doesn't work with keypering